driloader/browser/chrome.py

```diff
@@ -20,29 +25,45 @@ class Chrome(BaseBrowser):
     Implements all BaseBrowser methods to find the proper Chrome version.
     """
 
-    __instance = None
+    __installed_version = None
```

Comment:
I don't think we need to use double underscore here. Single underscore would be enough.